src/structures/Message.js

```diff
@@ -290,7 +290,7 @@ class Message extends Base {
   patch(data) {
     const clone = this._clone();
 
-    if ('edited_timestamp' in data) this.editedTimestamp = new Date(data.edited_timestamp).getTime();
+    if (data.edited_timestamp !== null) this.editedTimestamp = new Date(data.edited_timestamp).getTime();
```

Comment:
Since undefined is !== null, won't this possibly become NaN now?

Reply:
It will never be undefined though as the docs suggest

Reply:
Patching or creating a message with partial data may not contain that property at all.

Reply:
Fair enough, I'll change it to !data.edited_timestamp